docs/docusaurus/src/css/searchBar.scss

```diff
@@ -1,15 +1,19 @@
-.custom-search-bar .DocSearch-Button {
-  background: none;
+.custom-search-bar {
```

Comment:
This styling looks unrelated to the rest of the PR (same with the change in custom.scss). Would you mind splitting these changes out into a separate PR so this one can be focused only on the versioning concnerns?

Reply:
Actually these are not needed anymore by using the approach of `visibility: hidden` instead of `display: none`, so I'm taking them out!